src/components/CustomNavLink/CustomNavLink.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC, memo } from 'react';
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
+interface Props {
+  to: string,
```

Comment:
you can use "To" from React instead of "typ string for to. Vova said todau that it is more reliable.
to: To,